src/griffe/dataclasses.py

```diff
@@ -428,6 +432,22 @@ def is_kind(self, kind: str | Kind | set[str | Kind]) -> bool:
             kind = Kind(kind)
         return self.kind is kind
 
+    @cached_property
+    def inherited_members(self) -> dict[str, Alias]:
+        """Members that are inherited from base classes."""
+        if not isinstance(self, Class):
+            return {}
```

Comment:
Only classes have "inherited" members. Inherited members won't be populated for class instances (attributes). Attributes have a type that points to the relevant class.

This is up to discussion, especially regarding inspection with dynamic base classes.